cd/manager/aws/ecs.go

```diff
@@ -162,6 +162,10 @@ func (e Ecs) GenerateEnvLayout(component manager.DeployComponent) (*manager.Layo
 }
 
 func (e Ecs) componentTask(component manager.DeployComponent, cluster, service string) (*manager.Task, bool) {
+	// Skip any ELP nodes
+	if strings.Contains(service, manager.ServiceSuffix_Elp) {
```

Comment:
Since it's expected to be a suffix, and it's only 3 characters, how about `string.HasSuffix`?

Reply:
Updated to use `strings.Split`, as discussed.